src/fuzz/BUILD

```diff
@@ -0,0 +1,11 @@
+
+
+cc_binary(
+  name = "dummy_fuzz_test",
```

Comment:
Maybe "simple_fuzz_test"?

This is the simplest integration, but it /is/ a valid integration.